resources/user_config.js

```diff
@@ -16,17 +16,20 @@ const Responses = _Responses;
 import _ZoneId from './zone_id.js';
 const ZoneId = _ZoneId;
 
-let UserConfig = {
-  optionTemplates: {},
-  userFileCallbacks: {},
-  savedConfig: null,
-  registerOptions: function(overlayName, optionTemplates, userFileCallback) {
+
+class UserConfig {
+  constructor() {
+    optionTemplates = {};
+    savedConfig = null;
+    userFileCallbacks = {};
```

Comment:
You're missing a `this.` in front of each of these.

Reply:
Oops!